broker/broker-security-api/src/main/java/org/codice/ddf/broker/security/api/SecurityServerPlugin.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.broker.security.api;
+
+import org.apache.activemq.artemis.api.core.Message;
+import org.apache.activemq.artemis.core.server.ServerSession;
+import org.apache.activemq.artemis.core.transaction.Transaction;
+
+public interface SecurityServerPlugin {
+
+  void beforeSend(
+      ServerSession session,
+      Transaction tx,
+      Message message,
+      boolean direct,
+      boolean noAutoCreateQueue);
```

Comment:
✏️ I find boolean parameters easier to understand when they're framed like "do this thing (yes/no)" vs "don't do this thing (yes/no)". So instead of `noAutoCreateQueue` it would be `autoCreateQueue`.

Reply:
This is an interface from Artemis, I think we should leave it as `noAutoCreateQueue` for consistency.